assembl/configs/aws.yaml

```diff
@@ -15,7 +15,7 @@ attachment_bucket: "assembl-data-%(aws_client)s"
 elasticsearch_index: "%(aws_client)s:assembl-es"
 elasticsearch_port: 80
 elasticsearch_version: 5.6.8
-aws_secrets_id: assembl_secrets
+aws_secrets_ids: arn:aws:secretsmanager:eu-west-1:900208097337:secret:assembl_secrets-1eqvUR
```

Comment:
Is this real secret ?

Reply:
No, these are keys to fetch from SecretManager on AWS. No secrets shared on the filesystem